test/livebook_web/live/hub/new/enterprise_component_test.exs

```diff
@@ -0,0 +1,119 @@
+defmodule LivebookWeb.Hub.New.EnterpriseComponentTest do
+  use Livebook.EnterpriseIntegrationCase, async: true
+
+  import Phoenix.LiveViewTest
+
+  alias Livebook.Hubs
+
+  setup do
+    on_exit(&Hubs.clean_hubs/0)
```

Comment:
This may affect other tests running asynchronously, we should just make sure the tests are agnostic to additional hubs :)